### PR TITLE
Update set-up-multi-factor-authentication.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note on why changing user states is no longer recommended and steps to enable per user.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2064

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/admin/security-and-compliance/set-up-multi-factor-authentication.md [hunk 1]
```diff
@@ -60,6 +60,9 @@ All other users will be asked to perform additional authentication when needed.
 - Per-user account (not recommended)
 
   Users choose the additional verification method during MFA registration.
+  
+ > [!NOTE]
+ > Changing user states is no longer recommended unless your licenses don't include Conditional Access as it requires users to perform MFA every time they sign in. To know more about changing status per-user account, See [Change status of a user](https://docs.microsoft.com/azure/active-directory/authentication/howto-mfa-userstates#change-the-status-for-a-user)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/security-and-compliance/set-up-multi-factor-authentication.md]:
```suggestion
 > Changing user states is no longer recommended unless your licenses don't include Conditional Access as it requires users to perform MFA every time they sign in. To know more about changing status per-user account, see [Change status of a user](https://docs.microsoft.com/azure/active-directory/authentication/howto-mfa-userstates#change-the-status-for-a-user).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@Sirkkuw - PR has been copy edited and is ready for final review. This is based on issue #2064

Thank you.

**Comment by Sirkkuw** [comment]
I honestly don't understand what this means:
Changing user states is no longer recommended unless your licenses don't include Conditional Access as it requires users to perform MFA every time they sign in. To know more about changing status per-user account,

Can we make it little easier to parse:

If your subscription doesn't include Conditional Access changing user state is not recommended.... Plus what dies the idea of "change user state" mean?

**Pull request closed without merging**